source/includes/fact-connection-strings.rst

```diff
@@ -0,0 +1,16 @@
+:ref:`mongosync <c2c-mongosync>` uses a standard :ref:`MongoDB URI
+connection string <mongodb-uri>` to connect clusters. 
+
+The standard URI connection scheme has the form:
+
+.. code-block:: none
+
+   mongodb://[username:password@]host1[:port1][,...hostN[:portN]][/[defaultauthdb][?options]]
+
+Specify the hostnames of the :binary:`mongod` instances the same way
+that they are listed in your replica set configuration.
```

Comment:
Note this statement does not cover sharded clusters. In a sharded cluster, you will connect to the mongos

Reply:
Done